docs/examples/basic/README.md

```diff
@@ -0,0 +1,330 @@
+# Basic Tempo Example
+
+This notebook will walk you through an end-to-end example deploying a Tempo pipeline, running on its own Conda environment.
+
+## Prerequisites
+
+  * rclone and conda installed.
+  * Run this notebook within the `seldon-examples` conda environment. Details to create this can be found [here]().
+
+## Architecture
+
+We will show two Iris dataset prediction models combined with service orchestration that shows some arbitrary python code to control predictions from the two models.
+
+![architecture](architecture.png)
+
+
+```python
+from IPython.core.magic import register_line_cell_magic
+
+@register_line_cell_magic
+def writetemplate(line, cell):
+    with open(line, 'w') as f:
+        f.write(cell.format(**globals()))
+```
+
+## Train Iris Models
+
+We will train:
+
+  * A sklearn logistic regression model
+  * A xgboost model
+
+
+```python
+from sklearn import datasets
+from sklearn.linear_model import LogisticRegression
+import joblib
+iris = datasets.load_iris()
+X = iris.data  # we only take the first two features.
+y = iris.target
+logreg = LogisticRegression(C=1e5)
+logreg.fit(X, y)
+logreg.predict_proba(X[0:1])
+with open("./artifacts/sklearn/model.joblib","wb") as f:
```

Comment:
There's the problem if folder `./artifacts/sklearn` and `./artifacts/xgboost` does not exist. 
Probably good to add cell
```
%%bash
mkdir -p ./artifacts/sklearn
mkdir -p ./artifacts/xgboost
```
just before these.